Libraries/Microsoft.Teams.Apps/Contexts/Context.cs

```diff
@@ -41,7 +41,7 @@ public partial interface IContext<TActivity> where TActivity : IActivity
     /// <summary>
     /// the app logger instance
     /// </summary>
-    public ILogger Log { get; set; }
+    public ILogger<App> Log { get; set; }
```

Comment:
Instead of exposing `ILogger` in `App.Log`, each consuming class should be responsible to define it's own `ILogger<OfMe>`, to be able to filter by namespaces.

By redirecting all logs to App.Log seems we are missing the LogCategory concept from ILogger.